sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -148,11 +148,16 @@ static void send_wp_moved(struct transport_tx *trans, struct link_device *dev)
   static uint8_t i;
   i++;
   if (i >= nb_waypoint) { i = 0; }
+  struct EnuCoor_i wp_i;
```

Comment:
What's up with those changes?
They don't seem to change anything functionally, just add some extra copying...